.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like an accidental add, `.idea/` could be added to .gitignore

Reply:
Good catch, it looks like the .gitignore in this project doesn't have any additions to IDE based project files, so rather than bog it down with such, I've deleted the associated IDE files, and left the .gitignore as is.